frontend/src/App.tsx

```diff
@@ -1,13 +1,13 @@
-import React from 'react';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-import Navigation from './components/Navigation';
-import Home from './Home';
-import './styles/main.css';
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import SideBar from "./components/SideBar";
```

Comment:
Your linter has converted these all to double quotes rather than single quotes, pls fix (should be ' not ")